src/rollup/types.d.ts

```diff
@@ -0,0 +1,343 @@
+import { EventEmitter } from 'events';
+
+// ast node types just any, unless acorn public dependency added
```

Comment:
Hm, that's kind of a step back. As a plugin author I'm loosing information here since `Program` and `Node` are referenced (indirectly) in some of the hooks.